test/controllers/nonsipp/otherassetsdisposal/ReportedOtherAssetsDisposalListControllerSpec.scala

```diff
@@ -92,4 +137,97 @@ class ReportedOtherAssetsDisposalListControllerSpec extends ControllerBaseSpec {
 
     act.like(journeyRecoveryPage(onSubmit).updateName("onSubmit" + _))
   }
+
+  "ReportedOtherAssetsDisposalListController in view only mode" - {
+
+    val currentUserAnswers = defaultUserAnswers
```

Comment:
Nice tests! One thing I think you could do to cut down on duplicated code is replace this whole block with:

> val currentUserAnswers = completedUserAnswers
>       .unsafeSet(FbVersionPage(srn), "002")
>       .unsafeSet(CompilationOrSubmissionDatePage(srn), submissionDateTwo)